src/views/datasets/ma-buoy/summary.vue

```diff
@@ -4,13 +4,21 @@
       <p>
         This dataset spans from {{ store.minDate.slice(0, 4) }} to
         {{ store.maxDate.slice(0, 4) }} and was collected by the Narragansett
-        Bay Fixed Station Monitoring Network (NBFSMN) with MassDEP as the lead
-        agency. The heatmap below summarizes the number of observations
-        collected for each month for different variables. Use this heatmap to
-        help you decide what data you want to visualize or download. When you
-        have an idea, go ahead and select the buoys, variables and dates to
-        explore. Or download the data in the most appropriate format for your
-        analyses! To begin, select a variable to see what data is available.
+        Bay Fixed Station Monitoring Network (<a
+          href="http://www.dem.ri.gov/programs/emergencyresponse/bart/stations.php"
```

Comment:
would it be better to put the link around the long form instead of the abbreviation?

Reply:
or both?

Reply:
A lot of them only have an acronym, so maybe for consistency, just the acronyms?

Reply:
I'd maybe do it around the whole thing `<a ...>Agency Name (AN)</a>